android/java/org/chromium/chrome/browser/brave_news/CardBuilderFeedCard.java

```diff
@@ -76,6 +76,7 @@
 import java.util.concurrent.ExecutorService;
 import java.util.concurrent.Executors;
 
+@SuppressWarnings("checkstyle:SetTextColorAndSetTextSizeCheck")
```

Comment:
why are we suppressing this instead of setting them?

Reply:
cc: @AlexeyBarabash

Reply:
I am fixing it right now.
It requires ~50 lines to change and extract the affected lines into styles and dimensions xmls.

Reply:
Reverted at https://github.com/brave/brave-core/pull/28944/commits/cf55d68315efcb2370224497d48b0401fb4800a2 and fixed at https://github.com/brave/brave-core/pull/28944/commits/5bb879b937c27082d8041fc9f03f37239a3bb09e